### ISSUE TITLE
quiz question this week

### ISSUE THREAD

**Issue by mmiddleton** [sr320/course-fish546-2016#83]
I am having trouble getting my data file to load into CoGe as shown in the gif. I copied and pasted the url from the table but when I hit the next button I get an error telling me that I can only use certain file types which are listed as part of the error. I assume that this is because although the file is a fastq file, it is compressed so it doesn't recognize it as being a fastq file due to the .gz at the end of the file name. I downloaded the file using `curl -O [url]` and unzipped the file in an attempt to upload the file rather than use the url, but unzipped my file is 2.4GB and the recommendations/restrictions are files less than 2GB. Did anyone else have this problem and find a way around it? Or am I simply doing the exercise wrong?

**Comment by sr320** [comment]
Using the .gz files work

**Comment by mmiddleton** [comment]
I got the file to load (I didn't realize that the "go" button next to login was for getting the file and not for logging in, the hint gif only works on one of the three computers I have access to and this isn't one of them so I'm doing this from memory...). But now, when I got to the end to hit the "start" button I got this error message:
![error](https://cloud.githubusercontent.com/assets/22554467/20193667/e42d5dae-a743-11e6-9900-4e0a9bf2b305.png)
Is that maybe because there's a message on the homepage saying that CoGe may be unavailable for short periods today because of a hardware upgrade? Or is this a message you've seen before and I need to change something about the options that I selected?
